scripts/skills/actives/rf_shield_bash_skill.nut

```diff
@@ -0,0 +1,102 @@
+this.rf_shield_bash_skill <- ::inherit("scripts/skills/skill", {
+	m = {},
+	function create()
+	{
+		this.m.ID = "actives.rf_shield_bash";
+		this.m.Name = "Shield Bash";
+		this.m.Description = "Bash your shield into your opponent to push them off balance and disorient them.";
+		this.m.Icon = "skills/rf_shield_bash_skill.png";
+		this.m.IconDisabled = "skills/rf_shield_bash_skill_sw.png";
+		this.m.Overlay = "rf_shield_bash_skill";
+		this.m.SoundOnUse = [
+			"sounds/combat/knockback_01.wav",
+			"sounds/combat/knockback_02.wav",
+			"sounds/combat/knockback_03.wav"
+		];
+		this.m.SoundOnHit = [
+			"sounds/combat/knockback_hit_01.wav",
+			"sounds/combat/knockback_hit_02.wav",
+			"sounds/combat/knockback_hit_03.wav"
+		];
+		this.m.Type = ::Const.SkillType.Active;
+		this.m.Order = ::Const.SkillOrder.UtilityTargeted;
+		this.m.IsSerialized = false;
+		this.m.IsActive = true;
+		this.m.IsTargeted = true;
+		this.m.IsAttack = true;
+		this.m.HitChanceBonus = 25;
+		this.m.IsIgnoredAsAOO = true;
+		this.m.ActionPointCost = 4;
+		this.m.FatigueCost = 20;
+		this.m.MinRange = 1;
+		this.m.MaxRange = 1;
+		this.m.AIBehaviorID = ::Const.AI.Behavior.ID.KnockOut;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getDefaultUtilityTooltip();
+
+		if (this.m.HitChanceBonus != 0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Has " + ::MSU.Text.colorizeValue(this.m.HitChanceBonus, {AddSign = true, AddPercent = true}) + " chance to hit")
+			});
+		}
+
+		ret.push({
+			id = 11,
+			type = "text",
+			icon = "ui/icons/special.png",
+			text = ::Reforged.Mod.Tooltips.parseString("Will [daze|Skill+dazed_effect] the target on a hit")
+		});
+
+		return ret;
+	}
+
+	function onVerifyTarget( _originTile, _targetTile )
+	{
+		return this.skill.onVerifyTarget(_originTile, _targetTile) && !_targetTile.getEntity().getCurrentProperties().IsImmuneToDaze;
+	}
+
+	function onUse( _user, _targetTile )
+	{
+		local targetEntity = _targetTile.getEntity();
+
+		this.spawnAttackEffect(_targetTile, ::Const.Tactical.AttackEffectImpale);
+		if (!this.attackEntity(_user, targetEntity))
+			return false;
+
+		targetEntity.getSkills().add(::new("scripts/skills/effects/dazed_effect"));
+
+		if (!_user.isHiddenToPlayer() || _targetTile.IsVisibleForPlayer)
+		{
+			::Tactical.EventLog.log(::Const.UI.getColorizedEntityName(_user) + " has dazed " + ::Const.UI.getColorizedEntityName(targetEntity));
+		}
+
+		::Sound.play(::MSU.Array.rand(this.m.SoundOnHit), ::Const.Sound.Volume.Skill, targetEntity.getPos());
+
+		return true;
+	}
+
+	function onAfterUpdate( _properties )
+	{
+		if (_properties.IsProficientWithShieldSkills)
+		{
+			this.m.FatigueCostMult *= ::Const.Combat.WeaponSpecFatigueMult;
```

Comment:
No other shield skill has this and we don't mention this in the tooltip of shield expert

Reply:
All shield skills have this in vanilla. This is from the oath of fortification.